scripts/helpers.mjs

```
@@ -359,3 +362,28 @@ export const pfv = (type) => {
+
+// ======================================================================================
+// pfIcon: a helper function to use svg's from @patternfly/react-icons
+// ======================================================================================
+//
+// Usage:
+//   {{pfIcon 'arrow-right'}}
+//
+// ======================================================================================
+export const pfIcon = function (iconName) {
+  try {
+  if (!iconName || typeof iconName !== 'string') {
```

ESLint

[error] Expected indentation of 4 spaces but found 2.

(indent)